inductiva/tasks/output_consts.py

```diff
@@ -0,0 +1,43 @@
+"""Constants for connecting tasks to their simulation output classes."""
+import inductiva
+
+SCENARIO_METHODS_NAMES = [
```

Comment:
Can you think of a way of avoiding these names repeated 3 times (this list and the keys for the two dictionaries)? DRY principle.